packages/calypso-e2e/src/email-client.ts

```diff
@@ -136,7 +136,7 @@ export class EmailClient {
 	 * @throws {Error} IF the message did not have any links.
 	 */
 	getMagicLink( message: Message ): URL {
-		const link = message.text?.links?.pop();
+		const link = message.text?.links?.shift();
```

Comment:
The email template has been updated, and the position of the magic link is now in the first place.